src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/10 Page.md

```diff
@@ -0,0 +1,67 @@
+| [page.setViewportSize(viewportSize)](/javascript-api/k6-experimental/browser/page/setviewportsize/)       | Updates the page's width and height. |
+| [page.tap(selector, [options])](/javascript-api/k6-experimental/browser/page/tap/)       | Taps the first element that matches the selector.                                      |
+| [page.textContent(selector, [options])](/javascript-api/k6-experimental/browser/page/textcontent/)       | Returns the `element.textContent`.         |
+| [page.title()](/javascript-api/k6-experimental/browser/page/title/)       | Returns the page's title.         |
+| [page.type(selector, text, [options])](/javascript-api/k6-experimental/browser/page/type/)       | Types the `text` in the first element found that matches the selector.              |
+| [page.touchScreen()](/javascript-api/k6-experimental/browser/page/touchscreen/)       | Returns the [Touchscreen](/javascript-api/k6-experimental/browser/touchscreen/) instance to interact with a virtual touchscreen on the page.                                      |
+| [page.uncheck(selector, [options])](/javascript-api/k6-experimental/browser/page/uncheck/)       | Unselects an input checkbox.                                    |
+| [page.url()](/javascript-api/k6-experimental/browser/page/url/)       | Returns the page's URL.         |
+| [page.viewportSize()](/javascript-api/k6-experimental/browser/page/viewportsize/)       | Returns the page's size (width and height).         |
+| [page.waitForFunction(pageFunction, arg, [options])](/javascript-api/k6-experimental/browser/page/waitforfunction/)       | Returns when the `pageFunction` returns a truthy value.         |
+| [page.waitForLoadState(state, [options])](/javascript-api/k6-experimental/browser/page/waitforloadstate/)       | Waits for the given load state to be reached.         |
+| [page.waitForNavigation([options])](/javascript-api/k6-experimental/browser/page/waitfornavigation/)       | Waits for the given navigation lifecycle event to occur and returns the main resource response.         |
+| [page.waitForSelector(selector, [options])](/javascript-api/k6-experimental/browser/page/waitforselector/)       | Returns when element specified by selector satisfies `state` option.        |
+| [page.waitForTimeout(timeout)](/javascript-api/k6-experimental/browser/page/waitfortimeout/)       | Waits for the given `timeout` in milliseconds. |
+| [page.workers()](/javascript-api/k6-experimental/browser/page/workers/)       | Returns an array of the dedicated WebWorkers associated with the page. |
+| [page.$(selector)](/javascript-api/k6-experimental/browser/page/page-$/)       | Finds an element matching the specified selector within the page. |
+| [page.$$(selector)](/javascript-api/k6-experimental/browser/page/page-$$/)       | Finds all elements matching the specified selector within the page. |
```

Comment:
Can we move these two rows after the `dispatchEvent` one so the table order matches the method list in the side bar?